### PR TITLE
Fixes to configuration description and example.

### PR DESCRIPTION
**Pull request by Dav0815**
Alignment with new documentation guidelines (https://developers.home-assistant.io/docs/en/documentation_standards.html)

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/sensor.bom.markdown [hunk 1]
```diff
@@ -62,11 +60,73 @@ To get the station ID for any BOM station:
...
  - For Adelaide, the URL will look like `http://www.bom.gov.au/products/IDS60801/IDS60801.94675.shtml`; the station ID is `IDS60801.94675`.
 
-Configuration variables:
-
-- **station** (*Optional*): The station ID string as identified from the BOM website.  If not given, defaults to the closest station.
-- **name** (*Optional*): The name you would like to give to the weather station.
-- **monitored_conditions** (*Required*): A list of the conditions to monitor.
+{% configuration %}
+station:
+  description: The station ID string as identified from the BOM website.  If not given, defaults to the closest station based on location data in configuration.yaml.
```
Comment (WofWca) [inline source/_components/sensor.bom.markdown]: Is `default` key applicable here?

File: source/_components/sensor.bom.markdown [hunk 2]
```diff
@@ -62,11 +60,73 @@ To get the station ID for any BOM station:
...
+  required: false
+  type: string
+name:
+  description: The name you would like to give to the weather station.
+  required: false
+  type: string
+monitored_conditions:
+  description: A list of the conditions to monitor.
+  required: true
+  type: map
```
Comment (WofWca) [inline source/_components/sensor.bom.markdown]: Still don't know if it should be map or list.

File: source/_components/sensor.waqi.markdown [hunk 3]
```diff
@@ -26,15 +26,22 @@ sensor:
...
-- **stations** (*Optional*): A list of station names to look for air quality data. Station should be within locations specified above.
+{% configuration %}
+token:
+  description: The token for the AQICN public API.
+  required: true
+  type: string
+locations:
+  description: A list of location names to look for air quality data. In case a specific location has multiple registered stations all of them will be added to Home Assistant.
+  required: true
+  type: string, list
```
Comment (WofWca) [inline source/_components/sensor.waqi.markdown]: Not sure if it can be `string`.

File: source/_components/sensor.waqi.markdown [hunk 4]
```diff
@@ -26,15 +26,22 @@ sensor:
...
+stations:
+  description: A list of station names to look for air quality data. Station should be within locations specified above.
+  required: false
+  type: string, list
```
Comment (WofWca) [inline source/_components/sensor.waqi.markdown]: Here too

File: source/_components/sensor.waqi.markdown (hunk 3 above)
Comment (klaasnicolaas) [reply source/_components/sensor.waqi.markdown]: Think it must only be a list

File: source/_components/sensor.bom.markdown (hunk 1 above)
Comment (Dav0815) [reply source/_components/sensor.bom.markdown]: Makes sense

**Review by frenck — CHANGES_REQUESTED** [review]
Thank you @Dav0815!
However, this PR is targeted against the wrong branch. Updates to current documentation need to go into the `current` branch. Please update the target and rebase your changes onto the `current` branch.

**Pull request closed without merging**